ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppRescheduleVisitPage.java

```diff
@@ -0,0 +1,210 @@
+
+package org.motechproject.ebodac.uitest.page;
+
+import org.motech.page.AbstractBasePage;
+import org.openqa.selenium.*;
```

Comment:
Try not to use wildcard import, they are bad for maintainability of code. You can force intellij not to do it
http://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Checkstyle rule about this:
http://checkstyle.sourceforge.net/config_imports.html#AvoidStarImport